testpilot/frontend/static-src/app/main.js

```diff
@@ -79,6 +79,23 @@ app.extend({
     } else {
       hitCallback();
     }
+  },
+
+  // Subscribe to basket
+  subscribeToBasket(email, callback) {
+    const url = 'https://basket.mozilla.org/news/subscribe/';
+    fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded'
+      },
+      body: 'newsletters=test-pilot&email=' + email
+    }).then(function subscribeReturn() {
```

Comment:
Could this line maybe just be `}).then(callback)`? Or does the callback just absolutely have to receive no parameters?